### PR TITLE
Remove old trie storage

### PR DESCRIPTION
**Pull request by BeniaminDrasovean**
Remove the old implementation of trie storage (the one in which the db was saved in the Static folder) 
Remove the code that saved the total number of trie checkpoints (it was not used).

### CODE REVIEW

**Comment by KritsanaVeingin** [comment]
Tell me, with 2500 Egld still a little confused. Do i need to deposit
my friend to buy this amount of coins before doing every transaction
of egld before confirming that i understand i need to have enough
money to exchange tokens or nft guys That came back, right? Am I
right? And what should I do when My wallet hasn't been verified yet,
I'm still confused.

[image: Community Verified icon]

ในวันที่ ศ. 15 เม.ย. 2022 15:08 น. codecov[bot] ***@***.***>
เขียนว่า:

**Review by bogdan-rosianu — APPROVED** [review]

File: cmd/node/config/enableEpochs.toml [hunk 1]
```diff
@@ -156,9 +156,6 @@
     # FrontRunningProtectionEnableEpoch represents the epoch when the first version of protection against front running is enabled
     FrontRunningProtectionEnableEpoch = 1
 
-    # DisableOldTrieStorageEpoch represents the epoch when the old trie storage implementation will be disabled
-    DisableOldTrieStorageEpoch = 2
```
Comment (raduchis) [inline cmd/node/config/enableEpochs.toml:160]: is it ok to remove the DisableEpoch?

**Review by raduchis — APPROVED** [review]

File: cmd/node/config/enableEpochs.toml (hunk 1 above)
Comment (BeniaminDrasovean) [reply cmd/node/config/enableEpochs.toml:160]: Yes, because this "event" already happened on mainnet, so everybody will start without the old storage.

Comment (raduchis) [reply cmd/node/config/enableEpochs.toml:160]: ok

**Review by popenta — APPROVED** [review]
@@                            Log scanner                                  @@
##                            remove-old-trie-storage                      ##
================================================================================
- Known Warnings              173
- New Warnings                2
- Known Errors                1
- New Errors                  0
- Panics                      0
================================================================================

**Pull request merged**